packages/babel-plugin/src/utils/is-jsx-function.ts

```diff
@@ -0,0 +1,23 @@
+import type { NodePath } from '@babel/traverse';
+import * as t from '@babel/types';
+
+import type { State } from '../types';
+
+export const isJsx = (
```

Comment:
Ah so this is looking for the transformed jsx function? Can we call the function that?

```suggestion
export const isTransformedJsxFunction = (
```